include/ensmallen_bits/cmaes/cmaes_impl.hpp

```diff
@@ -99,8 +100,7 @@ typename MatType::elem_type CMAES<SelectionPolicyType>::Optimize(
 
   std::vector<BaseMatType> mPosition(2, BaseMatType(iterate.n_rows,
       iterate.n_cols));
-  mPosition[0] = lowerBound + arma::randu<BaseMatType>(
-      iterate.n_rows, iterate.n_cols) * (upperBound - lowerBound);
+  mPosition[0] = iterate;
```

Comment:
Should we apply the transformation policy here, just in case the initial iterate is outside the bounds?

Reply:
We certainly could, but it isn't required. I request you to refer the comment above this one.